### PR TITLE
Added test case for defect MAGN-6862

### PR DESCRIPTION
**Pull request by riteshchandawar**
### Purpose
To catch regression in future.
Test Case for defect http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6862
PRs for Revit 2015 #378 and 2016 branch #379. 
### Declarations

Check these iff you believe they are true

- [x] The code base is in a better state after this PR
- [x] The level of testing this PR includes is appropriate

### Reviewers

@Randy-Ma 

### FYIs

@ikeough 

**Note** Test case is passing with recent build.
image

### CODE REVIEW

**Comment by riteshchandawar** [comment]
Pr for Revit 2015 branch: https://github.com/DynamoDS/DynamoRevit/pull/378

File: test/Libraries/RevitIntegrationTests/BugTests.cs [hunk 1]
```diff
@@ -621,6 +621,46 @@ public void MAGN_6710()
             Assert.AreEqual(1, curves.Count);
         }
 
+        [Test, TestModel(@".\Samples\DynamoSample_2014.rvt")]
+        public void MAGN_6862_CrashWhenBackToBackFileOpen()
+        {
+            var model = ViewModel.Model;
+
+            OpenSampleDefinition(@".\Revit\Revit_Adaptive Component Placement.dyn");
```
Comment (Randy-Ma) [inline test/Libraries/RevitIntegrationTests/BugTests.cs]: I did not see the "Automatic" RunMode.

**Comment by riteshchandawar** [comment]
Hello @Randy-Ma 

I have updated file with RunType = "Automatic" and there is no RunCurrentModel(); method in my test, so it's simply opening file and as it has Run Automatic so, they are running as it is and I am validating the Geometry created in Revit, to make sure it is actually running.

I hope now this test case is in good shape to merge.

Thanks for your comments.

image

image

**Comment by Randy-Ma** [comment]
LGTM

**Pull request merged**